config/config.go

```diff
@@ -12,7 +12,7 @@ var Version string
 var Clock TimestampGenerator = RealTimestampGenerator{}
 
 // Path to Mist's "Livepeer" process that we shell out to for the transcoding
-const PathMistProcLivepeer = "/usr/bin/MistProcLivepeer"
+const PathMistProcLivepeer = "/usr/local/bin/MistProcLivepeer"
```

Comment:
is it necessary to hardcode the full path to binary? wouldn't go be able to exec them from `$PATH` value?

Reply:
If we can simplify this so that Go can exec MistProcLivepeer from `$PATH` instead of hardcoding a path here (which resulted in issues previously when the path that MistProcLivepeer was at changed), then that seems like a worthwhile improvement. @hjpotter92 mind tracking this is in an issue so this can be addressed?

Reply:
Actually looks like this is tracked here https://github.com/livepeer/catalyst/issues/172